src/exceptions/OpenJDK-Assembly-exception-2.0.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="OpenJDK-assembly-exception-1.0" name="OpenJDK Assembly exception 1.0">
+      <crossRefs>
+         <crossRef>http://openjdk.java.net/legal/assembly-exception.html
+         </crossRef>
+      </crossRefs>
+      <titleText>OpenJDK Assembly Exception</titleText>
```

Comment:
This is `<titleText>` outside of `<text>`, which is why you're getting “[This element is not expected][1]”.  You need to add wrapping `<text>` like 4af440270 did for `LLVM-exception`.

[1]: https://travis-ci.org/spdx/license-list-XML/jobs/357048314#L653